packages/back-end/src/routers/teams/teams.controller.ts

```diff
@@ -307,3 +333,125 @@ export const deleteTeamById = async (
 };
 
 // endregion DELETE /teams/:id
+
+// region POST /teams/:id/members
+
+/**
+ * POST /teams/:id/members
+ * Add users in the list to the team
+ * @param req
+ * @param res
+ */
+export const addTeamMembers = async (
+  req: AuthRequest<{ members: string[] }, { id: string }>,
+  res: Response<DeleteTeamResponse>
+) => {
+  const { org } = getOrgFromReq(req);
+  const { id } = req.params;
+  const { members } = req.body;
+
+  req.checkPermissions("manageTeam");
+
+  const team = await findTeamById(id, org.id);
+
+  if (!team) {
+    return res.status(400).json({
+      status: 400,
+      message: "Team does not exist. Cannot add members.",
+    });
+  }
+
+  await Promise.all(
+    members.map((member) => {
+      return addMemberToTeam({
+        organization: org,
+        userId: member,
+        teamId: team.id,
+      });
+    })
+  );
```

Comment:
Again, avoid doing Mongo queries within a loop.  If 20 members are being added to a team at once, much better to do a single large update call vs 20 small ones.